src/main/java/core/basesyntax/service/strategy/Balance.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.service.ActivitiesService;
+
+public class Balance implements ActivitiesService {

```

Comment:
Follow name convention. Fix it for all operation 
```suggestion
public class BalanceActivityHandler implements ActivitiesService {
```